components/style/properties/gecko.mako.rs

```diff
@@ -1338,17 +1361,7 @@ ${impl_style_struct(style_struct)}
 ${define_ffi_struct_accessor(style_struct)}
 % endfor
 
-lazy_static! {
-    pub static ref INITIAL_GECKO_VALUES: ComputedValues = ComputedValues {
-        % for style_struct in data.style_structs:
-           ${style_struct.ident}: style_structs::${style_struct.name}::initial(),
-        % endfor
-        custom_properties: None,
-        shareable: true,
-        writing_mode: WritingMode::empty(),
-        root_font_size: longhands::font_size::get_initial_value(),
-    };
-}
+static mut INITIAL_GECKO_VALUES: *mut ComputedValues = 0 as *mut ComputedValues;
 
```

Comment:
I have a feeling that unsafecell may be needed here. But I'm not sure of the exact semantics of `static mut`. `*mut` itself is fine to use this way, but I'm not sure if we're modifying thus safely. Any unsafety here is purely theoretical and would only come up through extremenly aggressive inlining though.

cc @eddyb what are the alias semantics of static mut?